types/ya-disk/index.d.ts

```diff
@@ -0,0 +1,154 @@
+// Type definitions for ya-disk 4.0
+// Project: https://github.com/RomiC/ya-disk#readme
+// Definitions by: Vladislav Veluga <https://github.com/vladislav805>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
```

Comment:
```suggestion
// Minimum TypeScript Version: 4.1
```
why do you need this? This limits package interop with other content

Reply:
Do you suggest replace literals with `string`?

Reply:
Yes, I'd do this myself (this is why Node DT is not a state of art TS even there is a bunch of DT ninjas adding types to Node DT). That's up to you as the owner though 🗡️ . I understand the intent of using the template type literal here, but this comes with the price.
Otherwise, everything is OK

Reply:
well done with types version, thx!